### ISSUE TITLE
Increase level requirements pre next season

### ISSUE THREAD

**Issue by trollavitus** [UncreatedStaff/UncreatedWarfare#170]
To explain this practically right now Specialist/L4 is a really easily obtained rank and it unlocks the AH1Z and jet. The issue with this is both teams are getting a lot of recently promoted specs running the jets and crashing them, or pilots who do not understand how to either use the pilot lock on or flares correctly.

**Comment by DeathFromAbove145** [comment]
I think with the exp restrictions next season, it will nerf the rank system.

**Comment by WoeiSiesta** [comment]
Separate division lvl system like the early s2 would be better
Example: 
You have to play infantry division to get xp and lvl up for unlock more kits (infantry)
Then, you have to play Armoured division (AD) to get xp and lvl up for unlock higher lvl vehicles
Scout car - APC - IFV - MBT 
And the last one Airforce Division (AF) 
You have to fly the Blackhawk or mi-8 to get xp to unlock attack helicopters and jets
 The current AFs(both team) are so retarded asf(Lisane crooks and nemezis as example)
:trollface:

**Comment by trollavitus** [comment]
Yeah I agree the separate system is a pre good idea as it let's people understand how to play their role it also forces people to dip their toes with the BH before flying jet or AH 
What I recommend is having AH unlock first because it's easier to use so people understand the basics of flying attack craft then unlock jets so veteran pilots are the only ones using them 

Same deal with armors start em with Stryker then have brad lav unlock at the same time then tank

**Comment by WoeiSiesta** [comment]
But, the xp farming issue 
This will cause some mf is farming xp by transferring the builds or ammo to the nearest fob instead of the useful fobs 
This were happened in the early s2 
Digro clan was farming xp to get higher lvl for unlocking attack helis and jet when the admins aren’t online or touching grass frfr 
Idk if there’s way to fix this issue

**Comment by trollavitus** [comment]
I mean that's an issue even now seeing what I've been doing to make credits is just mass fob production so I can continue buying jets and AH sustainably

**Comment by trollavitus** [comment]
The best fix is to make it so resupply doesn't actually give any credits or xp but make it so building gives a ton more

**Comment by benvalkin** [comment]
I think a simple limit to how many supplies can be on a FOB at once would help a lot.

Another idea I had is to make it so that you only get XP for resupplying a FOB once the ammo/build you supplied actually gets used... It wouldn't be too hard to implement this actually, we can make FOBs keep track of who has dropped build/ammo onto it and how much, then give out XP awards accordingly when somebody does /ammo or builds something.

**Comment by ToastyTimeZ** [comment]
There is no scout cars next season, remember :trollface:

**Comment by ToastyTimeZ** [comment]
Depending on how the quest experiments have been going, I'm sure if they've been successful that implementing a system of which over time you gain experience and then before being able to level up with that experience there could be a finishing quest that requires kills, captures, vehicular kills, etc., because at least in my opinion that'd be a strong and easy way to do it with no complications arising.
